rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -962,6 +962,76 @@ public void updateCollections(Transaction rskTx) throws IOException {
         updateFederationCreationBlockHeights();
     }
 
+    protected void processSvpFundTransactionUnsigned(Transaction rskTx) throws IOException, InsufficientMoneyException {
+        Optional<Federation> proposedFederationOpt = federationSupport.getProposedFederation();
+        if (!proposedFederationOpt.isPresent()) {
+            String message = "Proposed federation should be present when processing SVP fund transaction.";
+            logger.warn(message);
+            throw new IllegalStateException(message);
+        }
+        Federation proposedFederation = proposedFederationOpt.get();
+
+        Coin spendableValueFromProposedFederation = bridgeConstants.getSpendableValueFromProposedFederation();
+        BtcTransaction svpFundTransactionUnsigned = createSvpFundTransaction(proposedFederation, spendableValueFromProposedFederation);
+
+        provider.setSvpFundTxHashUnsigned(svpFundTransactionUnsigned.getHash());
+        settleReleaseRequest(svpFundTransactionUnsigned, rskTx.getHash(), spendableValueFromProposedFederation);
+    }
+
+    private BtcTransaction createSvpFundTransaction(Federation proposedFederation, Coin spendableValueFromProposedFederation) throws IOException, InsufficientMoneyException {
```

Comment:
```suggestion
    private BtcTransaction createSvpFundBtcTransaction(Federation proposedFederation, Coin spendableValueFromProposedFederation) throws IOException, InsufficientMoneyException {
```

Wdyt?

Reply:
well, the method returns a `BtcTransaction`. And the fund transaction is a btc transaction. I think adding the `Btc` part is redundant